front/app/api/baskets_ideas/useVoting.tsx

```diff
@@ -40,11 +41,21 @@ export const VotingContext = ({ projectId, children }: Props) => {
   );
 };
 
-const useVotingInterface = (projectId?: string) => {
+type UseVotingInterfaceProps = {
+  projectId?: string;
+  phaseId?: string;
+};
+
+const useVotingInterface = ({
+  projectId,
+  phaseId,
```

Comment:
Why do we need this extra parameter here? Is it in case there are multiple voting phases, or for something else?

Reply:
Yep indeed - for when a user might be viewing a past voting phase (and there are other voting phases in the project) :)